bpf/Makefile.bpf

```diff
@@ -26,6 +26,13 @@ LLC   ?= llc
 HOST_CC    ?= gcc
 HOST_STRIP ?= strip
 
+TARGETARCH := $(shell go env GOARCH)
+
+ifeq ($(TARGETARCH),arm64)
```

Comment:
This needs to be more of a special case, it should be controlled only by a variable that is used inside of the Docker build. So if someone running on an Arm machine and trying to build it natively, it should use the native `gcc` and `strip` and not the cross-compiler.
```suggestion
ifeq ($(IMAGE_CROSS_TARGET_PLATFORM),linux/arm64)
```

Reply:
What is going to set IMAGE_CROSS_TARGET_PLATFORM or are we expecting to update the Dockerfile `RUN` statement to set this as an `env`?

Reply:
You answered question in the next feedback item :joy: